groups/tests/test_managers.py

```diff
@@ -0,0 +1,108 @@
+from django.test import TestCase
+from incuna_test_utils.compat import Python2AssertMixin
+
+from . import factories
+from .. import models
+
+
+class TestGroupManager(Python2AssertMixin, TestCase):
+    def test_discussions(self):
+        discussion = factories.DiscussionFactory.create()
+        discussion_two = factories.DiscussionFactory.create()  # Has a different group
+
+        results = models.Group.objects.discussions()
+        self.assertCountEqual([discussion, discussion_two], results)
+
+    def test_comments(self):
+        comment = factories.TextCommentFactory.create()
+        comment_two = factories.TextCommentFactory.create()  # Has a different group
+
+        results = models.Group.objects.comments()
+        self.assertCountEqual([comment, comment_two], results)
+
+    def test_users(self):
+        comment = factories.TextCommentFactory.create()
+        comment_two = factories.TextCommentFactory.create()  # Has a different group
+        factories.UserFactory.create()  # Hasn't posted anything
+
+        results = models.Group.objects.users()
+        self.assertCountEqual([comment.user, comment_two.user], results)
```

Comment:
Nice one on suggesting the duplicates; it turns out `filter`ing a queryset can in fact make it bigger...